script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -28,15 +28,15 @@ I can feel myself crack a smile.
 -- Page 33, Offset 14888.
 -- アルクェイドの事を考えるとつい笑ってしまうのは、アイツが最後まで突拍子もないヤツだったからだ。
 // 最後まで dropped because hard to make it not sound like she's dead or something
-Whenever I think about her, I can't help but smile.
+I can't help but feel this way whenever I think about her.
  I guess it's because she was such an incredible character.
 }
 [sha:5a010c24a17359803d800f2bcd3a05e8e2c2126a]{
 -- Page 34, Offset 14889.
 -- 無邪気に笑ったり怒ったり、たまにとんでもなく怖かったりもしたけど、基本的にはいいヤツだった。
 There was an innocence in the way that she got happy or angry,
  and there were times when she was truly terrifying,
- but she was fundamentally a good person.
+ but she was a fundamentally good person.
```

Comment:
Looking over this again, it actually feels worse.
Rather use the version with commas
>but she was, fundamentally, a good person.